### PR TITLE
fix(schema): allow `additionalProperties`

### LINKED ISSUE #206: 1.1.0 invalid options
`      {
        test: /\.(png|jpg|gif)$/,
        use: [
          {
            loader: 'url-loader',
            options: {
              limit: 10240,
            },
          },
        ],
      }
`
An error occurred:
    File Loader Invalid Options
    options['limit'] should NOT have additional properties

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
### Noteable Changes
---

Allow `addtionalProperties` for `url-loader` fallback compat

### Issues
---

- Fixes #206

### CODE REVIEW

**Comment by d3viant0ne** [comment]
@michael-ciniawsky - Shipping this as we have a broken minor version on npm

File: appveyor.yml [hunk 1]
```diff
@@ -6,23 +6,24 @@ init:
...
     - nodejs_version: '4.3'
-      webpack_version: 2.6.0
+      webpack_version: latest
       job_part: test
 build: 'off'
 matrix:
   fast_finish: true
 install:
-  - ps: Install-Product node $env:nodejs_version x64
+  - 'ps: Install-Product node $env:nodejs_version x64'
```
Comment (d3viant0ne) [inline appveyor.yml]: Pull the single quotes off all the powershell commands

File: .eslintrc [hunk 2]
```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "webpack",
-  "rules": {
-    "no-undefined": 0
-  }
-}
+  "extends": "webpack"
+}
```
Comment (d3viant0ne) [inline .eslintrc]: Roll this back to `"no-undefined": 0`

**Comment by michael-ciniawsky** [comment]
kk mom :)

**Comment by d3viant0ne** [comment]
And don't forget to brush your teeth before bed .....

**Comment by michael-ciniawsky** [comment]
mom === just a moment  

Is this uncommon to use ? 🙃  😕

**Comment by d3viant0ne** [comment]
Evidently I'm just getting old ...

**Pull request merged**